examples/sph/run_sod.py

```
@@ -164,3 +188,43 @@ def setup(self):
+
+print("##### Test result #####")
+result = {
+    "t": float(t[-1]),
+    "l2_rho": l2_rho[-1],
+    "l2_v": l2_v[-1].tolist(),
+    "l2_P": l2_P[-1],
+    "resol": resol,
+}
+print(result)
+
+json.dump(result, open(f"{dump_folder}/test_result.json", "w"), indent=4)
```

Ruff (0.15.21)

[warning] Use a context manager for opening files

(SIM115)